styles/colors-gallium-night.less

```diff
@@ -69,6 +69,7 @@
 // ---------------------------------------
 @agate-checkbox-text-color: @agate-white;
 @agate-checkbox-bg-color: @agate-accent;
+@agate-checkbox-border-color: transparent;
```

Comment:
Also not necessary, and in fact, this entire section can be deleted as it matches what's already defined in the day skin (used as the inherited base for Gallium).